lib/utils.js

```diff
@@ -226,7 +204,7 @@ function forEach(obj, fn) {
     obj = [obj];
   }
 
-  if (isArray(obj)) {
+  if (Array.isArray(obj)) {
```

Comment:
Proof that `Array.isArray` is well supported and possibly more reliable since it doesn't depend on a string representation check:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#Browser_compatibility